### PR TITLE
V2 Checkout: Sepa Form

### PR DESCRIPTION
**Pull request by GHaberis**
## What are you doing in this PR?

This PR introduces 2 new components: `SepaFormContainer` and `SepaForm`. The `SepaFormContainer` component manages state and the rendering of the presentational `SepaForm` component.  The `SepaForm` component is a copy of the existing `SepaForm`. I've copied it and co-located it with the `SepaFormContainer`, when copying it I also stripped the inline validation  in the old component as this will be addressed in a follow-up PR.

This PR does not include:
- Client side validation on the SEPA form.
 - Embedding the SEPA form in the new checkout, this is dependant on the creation of a PaymentMethod component that will wrap this SEPA form that hasn't been built yet.

[**Trello Card**](https://trello.com/c/3Urmo2zT/759-v2-checkout-build-sepa-form-component)

### CODE REVIEW

**Comment by GHaberis** [comment]
[V2 Checkout - Build SEPA form component](https://trello.com/c/3Urmo2zT/759-v2-checkout-build-sepa-form-component)

File: support-frontend/assets/components/sepaForm/SepaForm.tsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+						optional={false}
+						hideLabel={false}
+						label="Bank account holder name"
+						maxLength={40}
+						value={accountHolderName ?? undefined}
+						onChange={(e) => updateAccountHolderName(e.target.value)}
+					/>
+				</div>
+
+				<div />
```
Comment (i-hardy) [inline support-frontend/assets/components/sepaForm/SepaForm.tsx:69]: Can we get rid of these empty divs?

File: support-frontend/assets/components/sepaForm/SepaForm.tsx [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+const headerStyles = css`
+	${headline.xxxsmall({
+		fontWeight: 'bold',
+	})}
+`;
+const fieldsContainerStyles = css`
+	margin-top: ${space[4]}px;
+	> * + * {
+		margin-top: ${space[3]}px;
+	}
```
Comment (i-hardy) [inline support-frontend/assets/components/sepaForm/SepaForm.tsx:25]: I'm not wild about this kind of complex selector. If we just need things in a column with regular space in between we can use the [Stack](https://guardian.github.io/source/?path=/docs/packages-source-react-components-stack--playground) component from Source.

**Review by i-hardy — APPROVED** [review]
I think this is okay to go in, but it'd be nice to clean up a few things carried over from the old form.

File: support-frontend/assets/components/sepaForm/SepaForm.tsx (hunk 1 above)
Comment (GHaberis) [reply support-frontend/assets/components/sepaForm/SepaForm.tsx]: Yep, done!

File: support-frontend/assets/components/sepaForm/SepaForm.tsx (hunk 2 above)
Comment (GHaberis) [reply support-frontend/assets/components/sepaForm/SepaForm.tsx]: Done!

**Pull request merged**